src/Moka/Proxy/Proxy.php

```diff
@@ -35,9 +35,9 @@ public function serve(): MockObject
 
     /**
      * @param array $methodsWithValues
-     * @return Proxy
+     * @return ProxyInterface
      */
-    public function stub(array $methodsWithValues): self
+    public function stub(array $methodsWithValues): ProxyInterface
```

Comment:
I had doubts about this in the past: should a class with a fluent interface return its actual type - to force the concrete implemention from now on - or the interface itself? I'd really like your opinions on this.